src/client-apps/nuxeo-drive/nuxeo-drive-release-notes.md

```diff
@@ -3,197 +3,108 @@ title: Release Notes
+This is the first release that officially supports the new macOS version: Big Sur.
 
-<i class="fa fa-long-arrow-right" aria-hidden="true"></i>&nbsp;More on JIRA ticket [NXDRIVE-2147](https://jira.nuxeo.com/browse/NXDRIVE-2147)
+<i class="fa fa-long-arrow-right" aria-hidden="true"></i>&nbsp;More on JIRA ticket [NXDRIVE-2270](https://jira.nuxeo.com/browse/NXDRIVE-2270)
 
-#### Direct Edit and Unusual Digests
-
-It is now possible to Direct Edit a document when the attached file has no digest or when the digest is not standard.
-
-{{#> callout type='warning'}}
-Note that such documents will loose the ability to check their integrity and thus the conflicts detection will be ineffective.
-{{/callout}}
+### Direct Transfer
 
-<i class="fa fa-long-arrow-right" aria-hidden="true"></i>&nbsp;More on JIRA ticket [NXDRIVE-2236](https://jira.nuxeo.com/browse/NXDRIVE-2236)
+#### Fixed Multi-Screen Centering
 
-#### Force Refetch Server Config
+When opening the settings of Direct Transfer window, it is centered on the screen. But that was not working well for multi-screens setup. This is now fixed and the window will be centered on the screen where the mouse cursor is located.
 
-The fixed issue here is that Direct Edit starts only when the server configuration has been retrieved first. So when the app started and there were connection issues (bad network, no VPN, ...) then there was no Direct Edit thread started.
+<i class="fa fa-long-arrow-right" aria-hidden="true"></i>&nbsp;More on JIRA ticket [NXDRIVE-2353](https://jira.nuxeo.com/browse/NXDRIVE-2353)
 
-Later, still while the app was running, even if the network issues were fixed, it was not possible to use Direct Edit without restarting the app.
+#### Sessions for Direct Transfer
 
-Now the server configuration is fetched "on-demand" before doing any Direct Edit.
+It is now possible for the user to see and interact with the Direct Transfer sessions through the GUI. Two dedicated tabs have been added to the Direct Transfer window to show sessions statuses. These tabs list ongoing and completed sessions with some useful informations about them. It is also possible to pause/resume/cancel an ongoing session through this interface. The previously existing list of transferred items has been moved to a third tab called _Monitoring_.
 
-<i class="fa fa-long-arrow-right" aria-hidden="true"></i>&nbsp;More on JIRA ticket [NXDRIVE-2194](https://jira.nuxeo.com/browse/NXDRIVE-2194)
+The Monitoring tab now only shows transfers uploaded by chunks ([NXDRIVE-2432](https://jira.nuxeo.com/browse/NXDRIVE-2432)). And it is no more possible to pause a single transfer from that view, you will have to pause the session instead ([NXDRIVE-2433](https://jira.nuxeo.com/browse/NXDRIVE-2433)).
 
-### S3 Direct Upload
+<i class="fa fa-long-arrow-right" aria-hidden="true"></i>&nbsp;More on JIRA ticket [NXDRIVE-2309](https://jira.nuxeo.com/browse/NXDRIVE-2309)
```

Comment:
What about the warning about duplicates? Should we display something @aescaffre?